config/dynamicconfig/development_es.yaml

```diff
@@ -1,11 +1,9 @@
 frontend.enableClientVersionCheck:
   - value: true
-system.advancedVisibilityWritingMode:
-  - value: "on"
-system.advancedVisibilityMigrationWritingMode:
-  - value: "source"
-system.enableReadVisibilityFromES:
-  - value: true
+system.writeVisibilityStoreName:
+  - value: "es"
+system.readVisibilityStoreName:
+  - value: "os"
```

Comment:
Why do we write to es and read from os?

Reply:
Ha, I will update it

Reply:
I see,  this is the dynamic config, it was used for test. I will revert it when I finish test